sys/net/gnrc/network_layer/ipv6/ext/frag/Kconfig

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2019 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_GNRC_IPV6_EXT_FRAG
+    bool "Configure GNRC IPv6 extension header"
```

Comment:
The menuconfig thingy above says "...GNRC_IPV6_EXT_**FRAG**" (as does the PR description). So is this config section about [*all* extension headers](https://tools.ietf.org/html/rfc8200#section-4) or just [fragment headers](https://tools.ietf.org/html/rfc8200#section-4.5)?

Reply:
Right, the prompt is wrong here. Given the location of the file it is only for fragment header options.

Would `"Configure GNRC IPv6 extension fragment header"` OK?

Reply:
> Would `"Configure GNRC IPv6 extension fragment header"` OK?

The "official" name is just "IPv6 fragment header". If you want to have "extension header" in there for some reason, "IPv6 fragment extension header" would be more correct. However, the config options in this file don't really configure the fragment header, but IPv6 fragmentation and reassembly, which uses the fragment header. So "Configure GNRC IPv6 fragmentation and reassembly" would be the most accurate.

Reply:
Changed the name